Description: Execute shell with full script path, not just basename.
Origin: http://sources.redhat.com/ml/libc-alpha/2005-04/msg00080.html
Upstream status: Accepted, committed in CVS.

--- glibc-2.3.5/posix/execvp.c.orig	2005-02-26 02:02:17.000000000 +0100
+++ glibc-2.3.5/posix/execvp.c	2005-04-14 17:42:22.000000000 +0200
@@ -140,7 +140,7 @@
 	    {
 	      if (script_argv == NULL)
 		{
-		  script_argv = allocate_scripts_argv (file, argv);
+		  script_argv = allocate_scripts_argv (startp, argv);
 		  if (script_argv == NULL)
 		    {
 		      /* A possible EACCES error is not as important as
